agent-install/edgeNodeFiles.sh

```diff
@@ -175,12 +196,30 @@ function cleanUpPreviousFiles() {
     echo
 }
 
-# Get the edge cluster agent image tar file from the mgmt hub installation content
+# Pull the edge cluster agent image tar file either from a specified entitled registry or the openhorizon docker hub 
```

Comment:
Remove "entitled" from this comment, because it open source that doesn't have much meaning.